packages/tools/src/utilities/cine/playClip.ts

```diff
@@ -0,0 +1,246 @@
+import {
+  utilities,
+  getEnabledElement,
+  StackViewport,
+} from '@cornerstonejs/core';
+import CINE_EVENTS from './events';
+import { addToolState, getToolState } from './state';
+
+const { triggerEvent } = utilities;
+
+/**
+ * Starts playing a clip or adjusts the frame rate of an already playing clip.  framesPerSecond is
+ * optional and defaults to 30 if not specified.  A negative framesPerSecond will play the clip in reverse.
+ * The element must be a stack of images
+ * @param element - HTML Element
+ * @param framesPerSecond - Number of frames per second
+ */
+function playClip(element: HTMLDivElement, framesPerSecond: number): void {
+  let playClipData;
+  let playClipTimeouts;
+
+  if (element === undefined) {
+    throw new Error('playClip: element must not be undefined');
+  }
+
+  const enabledElement = getEnabledElement(element);
+
+  if (!enabledElement) {
+    throw new Error(
+      'playClip: element must be a valid Cornerstone enabled element'
+    );
+  }
+
+  const { viewport } = enabledElement;
+
+  if (!(viewport instanceof StackViewport)) {
+    throw new Error(
+      'playClip: element must be a stack viewport, volume viewport playClip not yet implemented'
+    );
+  }
+
+  const stackData = {
+    currentImageIdIndex: viewport.getCurrentImageIdIndex(),
+    imageIds: viewport.getImageIds(),
+  };
+
+  const playClipToolData = getToolState(element);
+
+  if (!playClipToolData) {
+    playClipData = {
+      intervalId: undefined,
+      framesPerSecond: 30,
+      lastFrameTimeStamp: undefined,
+      frameRate: 0,
```

Comment:
`frameRate` doesn't appear to be used? Should we remove it? It seems like a duplicate of `framesPerSecond`?